sip-servlets-client/src/main/java/org/mobicents/javax/servlet/sip/dns/DNSResolver.java

```diff
@@ -60,7 +61,7 @@
 	 * @param uri the uri used to find the corresponding SipURI
 	 * @return the SipURI found through ENUM methods or the uri itself if the uri is already a SipURI without a user=phone param
 	 */
-	SipURI getSipURI(URI uri);
+	SipURI getSipURI(URI uri, SipSession sipSession);
```

Comment:
Hi @jaimecasero ,

I have spent time to investigate how to use thread local in sipservlet, but still don't have any idea. Could you please help me to give me some suggestion about this issue.